blockchain/utxobackend.go

```diff
@@ -39,12 +97,21 @@ type UtxoBackend interface {
 	// both the entry and the error.
 	FetchEntry(outpoint wire.OutPoint) (*UtxoEntry, error)
 
+	// FetchInfo returns versioning and creation information for the UTXO backend.
+	FetchInfo() (*utxoBackendInfo, error)
```

Comment:
As called out elsewhere, this should be an exported type since it's part of an interface.

Reply:
Updated.